_maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm

```diff
@@ -96,9 +96,6 @@
 /obj/effect/decal/cleanable/dirt,
 /turf/open/floor/plasteel/floorgrime,
 /area/ruin/unpowered/syndicate_lava_base/chemistry)
-"cO" = (
-/turf/closed/wall/mineral/plastitanium/explosive,
```

Comment:
uuh are these meant to stay around?

Reply:
yes. these are what makes the self destruct work.

Reply:
assuming you used mapmerge2, it's fine. they're just unused keys being removed from the dictionary.

Reply:
I did use mapmerge2

Reply:
tis fine then